src/diffusers/utils/dummy_transformers_and_onnx_objects.py

```diff
@@ -0,0 +1,11 @@
+# This file is autogenerated by the command `make fix-copies`, do not edit.
+# flake8: noqa
+
+from ..utils import DummyObject, requires_backends
+
+
+class StableDiffusionPipeline(metaclass=DummyObject):
```

Comment:
I still don't fully understand how these tricks work, so excuse me for the noise. Do we need a dummy `StableDiffusionOnnxPipeline` too?

Reply:
Ah, didn't run `make fix-copies` lately, should be fixed now